src/main/java/org/isf/orthanc/manager/OrthancBrowserManager.java

```diff
@@ -0,0 +1,164 @@
+		return orthancUserIoOperation.updateOrthancUser(orthancUser);
+	}
+
+	/**
+	 * Return {@link OrthancPatient}
+	 * 
+	 * @param patId - the patient id
+	 * @return {@link OrthancPatient}. It could be {@code null}.
+	 * @throws OHServiceException 
+	 */
+	public OrthancPatient getOrthancPatientByPatientId(int patId) throws OHServiceException {
+		return orthancPatientIoOperation.getOrthancPatientByPatientId(patId);
+	}
+	
+	/**
+	 * Return {@link OrthancPatient}
+	 * 
+	 * @param id - the orthan patient id
+	 * @return {@link OrthancPatient}. It could be {@code null}.
+	 * @throws OHServiceException 
+	 */
+	public OrthancPatient getOrthancPatientById(int id) throws OHServiceException {
+		Optional<OrthancPatient> optionalOrthancPatient = orthancPatientIoOperation.getOrthancPatientById(id);
+		if (optionalOrthancPatient.isPresent()) {
+			return optionalOrthancPatient.get();
+		}
+		return null;
+	}
+	
+	/**
+	 * Insert an {@link OrthancPatient} in the DB
+	 * 
+	 * @param orthancPatient - the {@link OrthancPatient} to insert
+	 * @return the {@link OrthancPatient} that has been inserted, {@code null} otherwise.
+	 * @throws OHServiceException 
+	 */
+	public OrthancPatient newOrthancPatient(OrthancPatient orthancPatient) throws OHServiceException {
+		return orthancPatientIoOperation.newOrthancPatient(orthancPatient);
+	}
+	
+	/**
+	 * Update an {@link OrthancPatient}
+	 * 
+	 * @param orthancPatient - the {@link OrthancPatient} to update
+	 * @return the {@link OrthancPatient} that has been updated, {@code null} otherwise.
+	 * @throws OHServiceException 
+	 */
+	public OrthancPatient updateOrthancPatient(OrthancPatient orthancPatient) throws OHServiceException {
+		return orthancPatientIoOperation.updateOrthancPatient(orthancPatient);
+	}
+
+	public String getStudies() {
+		HttpHeaders headers = createHeaders("u2g", "u2g123"); // use the credential of current user here
+        HttpEntity<String> entity = new HttpEntity<>(headers);
+        RestTemplate restTemplate = createRestTemplate();
+        ResponseEntity<String> response = restTemplate.exchange(ORTHANCBASEURL + "/studies", HttpMethod.GET, entity, String.class);
+        return response.getBody();
+    }
+
+    private HttpHeaders createHeaders(String username, String password) {
+        return new HttpHeaders() {
+            {
+            String auth = username + ":" + password;
+            byte[] encodedAuth = Base64.getEncoder().encode(auth.getBytes(StandardCharsets.US_ASCII));
+            String authHeader = "Basic " + new String(encodedAuth);
+            set("Authorization", authHeader);
+            }
+        };
+    }
+    
+    private RestTemplate createRestTemplate() {
+    	RestTemplate restTemplate = new RestTemplate();
+        HttpComponentsClientHttpRequestFactory factory = new HttpComponentsClientHttpRequestFactory();
+        restTemplate.setRequestFactory(factory);
+        return restTemplate;
+    }
+}
```

Comment:
This is out of the scope of theses ticket. I think you should better focus on the the tests (Orthanc User and Orthan Patient CRUD)